### ISSUE TITLE
Feature Request: Predefined command groups/tasks/tabs

### ISSUE THREAD

**Issue by JWess** [felixse/FluentTerminal#851]
I am a developer. Every day I use FluentTerminal to fire up the same three processes in the same three directories. It would be really handy if I could define these as one command group. Then I could just click on the command group, and three tabs open and kickoff the three processes. Bonus points for being able to bind a command group to a keyboard shortcut.

**Comment by skipmeister123** [comment]
Why not create a script that launches those tabs? Then, create a profile that runs that script, and assign a keyboard shortcut.

**Comment by JWess** [comment]
> Why not create a script that launches those tabs? Then, create a profile that runs that script, and assign a keyboard shortcut.

Is there an example of this somewhere?

**Comment by JWess** [comment]
> > Why not create a script that launches those tabs? Then, create a profile that runs that script, and assign a keyboard shortcut.
> 
> Is there an example of this somewhere?

And would that be able to open them all as tabs in the same window?

**Comment by skipmeister123** [comment]
Yes, it can.  Please refer to the wiki. https://github.com/felixse/FluentTerminal/wiki/Command-Line-Arguments [https://github.com/felixse/FluentTerminal/wiki/Command-Line-Arguments]

**Comment by felixse** [comment]
https://github.com/felixse/FluentTerminal/issues/847 contains a nodejs script that does something similar
